examples/config/config

```diff
@@ -516,7 +517,7 @@ set ssl_policy fail
 
 # === Post-load misc commands ================================================
 spawn_sync_exec @scripts_dir/load_cookies.sh
-spawn_sync_exec @scripts_dir/load_cookies.sh @data_home/session-cookies.txt
+spawn_sync_exec @scripts_dir/load_cookies.sh @(echo "${UZBL_SESSION_COOKIE_FILE}")@
```

Comment:
It suppose the script examples/data/scripts/util/uzbl-dir.sh has been sourced somewhere.

The same assumption is made at the beginning of this file. The setting relying on
prefix, data_home, data_dirs, cache_home and config_home don't work either if
the XDG environment variables are not set.

For instance, the ctrl-m binding (adding a bookmark) won't work if data_home is not correctly set, which is the case if XDG_DATA_HOME is not correctly set, which could be the case if examples/data/scripts/util/uzbl-dir.sh is not sourced.